sdk/nodejs/src/client.ts

```diff
@@ -51,7 +51,8 @@ export class DVCClient {
                 this.eventQueue = new EventQueue(
                     this.logger,
                     environmentKey,
-                    options
+                    options,
+                    options?.reporter
```

Comment:
If we are already passing in `options` why can't we just read the `reporter` from there?

Reply:
Yeah I was just looking at this. The same applies for the logger.

Reply:
Passing both through options now